README.md

```diff
@@ -903,6 +903,46 @@ Running Tests:
 
 - `$ bundle exec rake` Run with `SKIP_FLAKY_TESTS=true` to skip flaky tests (CI runs all tests)
 
+### Interactive Debugging
```

Comment:
Can you name this "Interactive Test Debugging"? because it specifically allows you to run a debugger on tests